code/controllers/configuration.dm

```diff
@@ -360,6 +361,8 @@
 					config.rulesurl = value
 				if("githuburl")
 					config.githuburl = value
+				if("githubrepoid")
+					config.githuburl = value

```

Comment:
We can't convert it to a num here because then it becomes an exponential and the url won't work properly